tests/E2E/Cli/CheckCommandTest.php

```diff
@@ -48,8 +48,8 @@ public function test_app_returns_error_with_multiple_violations(): void
   should implement ContainerAwareInterface because all controllers should be container aware
 
 App\Domain\Model has 2 violations
-  depends on App\Services\UserService, but should not depend on classes outside namespace App\Domain because we want protect our domain (on line 14)
-  depends on App\Services\CartService, but should not depend on classes outside namespace App\Domain because we want protect our domain (on line 15)';
+  depends on App\Services\CartService, but should not depend on classes outside namespace App\Domain because we want protect our domain (on line 15)
+  depends on App\Services\UserService, but should not depend on classes outside namespace App\Domain because we want protect our domain (on line 14)';
```

Comment:
this shows something interesting. the `<=>` checks things in order as the properties appear in the class (afaik), so the error message is "more important" than the line number.

Reply:
i reverted this change and changed the Violation class to have the error after the line number.